### PR TITLE
feat: Add frontend ui

### LINKED ISSUE #25: Implement UI
Create a creative ui

### PR DESCRIPTION
**Pull request by coolusaHD**
Closes #25

### CODE REVIEW

File: backend/app.py
Comment (ghost) [inline backend/app.py:1]: Did we somehow change the port to 4000?

File: example.frontend.env
Comment (ghost) [inline example.frontend.env:1]: Any reason for another .env file? We could include the variables in the current example.env and use it for both front and backend if it does not make any difference; in this way we reduce one file.

File: backend/app.py
Comment (coolusaHD) [reply backend/app.py:1]: i think it was the default from my vite-ts-template
I think diffrent ports like 4000 and 5000 are better than 5001 and 5000 as you can differ between them eseaier

File: example.frontend.env
Comment (coolusaHD) [reply example.frontend.env:1]: vite needs the env file in the parent folder i think

File: backend/app.py
Comment (ghost) [reply backend/app.py:1]: Okay, then we might need to update the documentation and port in the docker to avoid mess. Could discuss this later!

**Review by ghost — APPROVED** [review]

**Pull request merged**